src/app/shared/services/user.service.ts

```diff
@@ -54,7 +54,7 @@ export class UserService {
     // Remove JWT from localstorage
     this.jwtService.destroyToken();
     // Set current user to an empty object
-    this.currentUserSubject.next(new User());
+    this.currentUserSubject.next(null);
```

Comment:
I'm reviewing here from my iPad, so bear with me 😅. but doesn't this change the behavior. before a new object has been passed in & now a null value instead.

also, in case it's the right approach I'd remove the old comment or rename it

Reply:
Yup that should be `.next({} as User)`

Reply:
done